src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1100,15 +1107,28 @@ public bool ApplyOrganelleUpgrade(OrganelleUpgradeActionData actionData)
                 actionData)));
     }
 
-    protected override int CalculateCurrentActionCost()
+    /// <summary>
+    ///   Adjusts the given cost by a multiplier and clamps it to a minimum of
+    ///   <see cref="Constants.MINIMUM_MUTATION_POINTS_COST"/> and a maximum of
+    ///   <see cref="Constants.BASE_MUTATION_POINTS"/>.
+    /// </summary>
+    /// <param name="cost">The action cost.</param>
+    /// <returns>The adjusted cost.</returns>
+    protected float GetAdjustedCost(float cost)
+    {
+        return Mathf.Clamp(
+            cost * CostMultiplier, Constants.MINIMUM_MUTATION_POINTS_COST, Constants.BASE_MUTATION_POINTS);
```

Comment:
Maybe this is the cause that breaks the mutation point math.

Reply:
This is required to cap MP at a minimum as hh requested, a fix in spite of this is needed.